github.md

```diff
@@ -22,3 +32,73 @@ you should do:
 ```bash
 $ git merge <branch-name> --no-ff
 ```
+
+### Pull Request Reviews
+
+In most (all) cases, pull requests should be reviewed by at least one other developer, even if it is just for syntax or raise general questions in the case of unfamiliarity with a particular language. 
+
+When creating a new pull request, you need to pick what branch you're merging - usually your dev branch - and what you're aiming to merge into - usually master, create a logical title, and add a description that includes: 
+* required: an explanation of what the reason/function of the code in the PR (the "why")
+* required: an explanation of what code changed in the PR (the "what")
+* optional: Link(s) to view changes, associated JIRA tickets, or other links that may be useful for understanding the changes in the PR.
+* optional: mention any person who may need to be aware of the changes in the PR (PM, UX, BO, or other devs)
+
+You should then assign one or more reviewers. (If you have more than one, you may want to mention in the description what each should be reviewing).
+
+Other notes: 
+"WIP" indicates a PR that is a work-in-progress that is not ready for review, but can be useful for comparing diffs as you work, or sharing in-progress work with others to see or comment on. In general, WIP PRs can be ignored.
+
+## Github and JIRA
+
+In general, if you are creating code on GitHub, commenting and review should happen there, with the code. If you are also using JIRA, you should add a comment with a link to the PR to the story in JIRA and move the story to the Review column to indicate that the PR is under review.
+
+(If changes are not commited to GitHub - like with Wordpress configurations - the comments and review should happen in JIRA; Mention reviewers by name in a comment on the ticket, what they should be reviewing, where they can view the changes, and then move the story to the Review column.)
+
+- - -
+
+## GitHub basics
+
+This process assumes you have: a terminal program (like iTerm with oh-my-zsh), an editor program (like Sublime Text or Atom), a github account and access to the MIT Libraries repos, and have already cloned the repos to your local machine. (For help with those steps, contact one of the MITLib devs.)
+
+*Start fresh*: Checkout master branch
+```git checkout master```
+
+*Get latest*: Get the latest changes that have been merged 
+```
+git fetch
+git merge
```

Comment:
Hmmm. I never do a `git merge` at this step. I instead `git checkout origin/master` as I prefer to not even have a local `master` branch because it's more trouble than it's worth.